docs/UserGuide.md

```diff
@@ -758,15 +760,15 @@ If you do not like an alias you originally set, you can use `unalias` to remove
     <th width="90" align="left">Input 1</th>
     <td >
 
-`unalias al/MyList`
+`unalias al/ml`
 
 </td>
   </tr>
   <tr align="left">
     <th>Output 1</th>
     <td>
 
-Removed alias 'myList'
+Removed alias 'ml'
```

Comment:
In my opinion, I actually would prefer the more verbose `myList` over `ml`, to make it clearer to the reader to understand that it's their own alias they are creating.

There is also still a discrepancy up in the Alias section (lines 731-733), where the example is `ml` and the paragraph below has `myList`.

Reply:
Owen mentioned this to me before, he thinks ml is better because it's shorter than the actual command 'list', especially since we say "The `alias` command can help you to shorten input commands to whatever you want it to be."